src/index.ts

```diff
@@ -35,8 +35,16 @@ const getRev = (osVersion: string) => {
 	}
 };
 
-const isDevelopmentVersion = (version: string) => {
-	return /(\.|\+|-)dev/.test(version);
+const getPreReleaseVariant = (version: string) => {
+	const matches = /-(dev|prod)/.exec(version);
```

Comment:
I'm not sure this is a great solution, mainly because we've never released a version number that looks like this, so it's never going to do anything. Good list of real world versions here: http://img.resinstaging.io/api/v1/image/raspberrypi3/versions. I'd expect that normally prerelease variants would come after some kind of specifier: `1.2.3-beta.dev`, similarly to `+rev4.dev`.